test/test-services.env

```diff
@@ -0,0 +1,60 @@
+# Any of these can be overridden locally in ../local.env
+
+# == TEST SERVICES
+
+# wikibase / wikibase-client / wikibase-jobrunner
+WIKIBASE_URL=http://wikibase.svc
```

Comment:
`*_SERVER` URLs all are now named `*_URL` and always have a protocol in front of them. These env vars are only used in the test environment. * In this case `MW_` is also renamed to `WIKIBASE_` which I find a bit more explicit in our context.

Reply:
Is this then a breaking change?

Reply:
Nope, these variables are only used within the test environment and have no impacted outside of it (e.g. in our Build setup, Docker images, or Example setup)